README.md

```diff
@@ -35,28 +35,37 @@ not present any other legal issues.
 * OSI approved license are strongly preferred.
 * Mods do not need to be minetest-licensing compatible (meaning, (A)GPL3.0+ is fine).
 * Non-OSI licenses may be acceptable, but this is determined on a one-by-one case.
-* Pure Public Domain is not acceptable, however WTFPL, MIT or similar usually is acceptable.
+* Pure Public Domain is not acceptable, however ISC, MIT or similar usually is acceptable.
+* WTFPL is expressly discouraged and we ask that you relicense to another license
+  before submitting. There are several appropriate alternatives, such as MIT, ISC etc..
 
 The project admins reserve the right to discontinue, remove or refuse any project based
-on legality, content or other, at their discretion.
+on legality, content or other, at their discretion. However, removal requests for other
+than reasons other than non-acceptable content or licensing are unlikely to be granted.
 
 The original author of the mod should be the one submitting it to the minetest-mods
 project. Only in exigent circumstances will a mod fork be allowed, and admins reserve
 the right to refuse mod forks at their discretion. In general, forks are only allowed
 if the original author clearly no longer maintains the project, and if the project has
 been properly licensed by the original author, and last, that the project license has
-remained unchanged in the fork.
+remained unchanged in the fork. For the purpose of determining whether a mod is a fork
+or an original work, we note that forks that were created from non-public sources where
+the original work is not publicly available are not considered forks, but are considered
+the original public works and acceptable.
 
 Modpacks (groups of mods bundled together) are not permitted in minetest-mods, as they
 make management more difficult and prevent users from installing only parts of of a
 modpack. For cooperative maintenance of modpacks, see the minetest-modpacks team instead,
 or look at the minetest-games project. (At the time of writing of this document, those
-two teams/projects do not exist).
+two teams/projects do not exist). A few modpacks exist in minetest-mods, but they are
+the only ones and new modpacks will not be allowed.
```

Comment:
while those are technically modpacks, they are so for organisational reasons (mesecons comes to mind)
they aren't modpacks in the traditional sense: collections of unrelated mods to provide a complete experience for users

maybe clarify that or something

Reply:
I prefer not to, because providing a technical exception case as to why modpacks are allowed will not be helpful in the long term I think, as people tend to look for excuses.

Having a historical cut-off (no more from now on) is a good way of saying "these mods were used on dozens of servers for years and we don't want them to become stale" while still being able to say "new modpacks should preferably be split up or made into a single mod" to new mods and bring the argument that those new mods aren't widely used yet so it's a better time..

All of this is also "at our discretion" as well. We can still make exceptions.

Reply:
I don't really like to idea of outright dismissing modpacks.  We can reject modpacks that have a bunch of unrelated stuff, but splitting mods up into modpacks can be useful for organization (eg, Technic and Mesecons and previously mentioned).

It seems like this would only encourage modders to combine a modpack into one huge mod, which isn't really any better.